conf/requirements.txt

```diff
@@ -3,3 +3,4 @@ termcolor
 tqdm
 yapf
 requests
+meson
```

Comment:
why do we need meson here? we have it in third_party/python

Reply:
Removed.